pkgs/dart_tooling_mcp_server/README.md

```diff
@@ -13,29 +13,20 @@ and use the compiled path as the command in your MCP server config.
 dart compile exe bin/main.dart
 ```
 
-### With the example DashBot
-
-After compiling the binary, you can run the example dash chat bot to interact
-with it, from the root of this package, after compiling the binary, run:
-
-```dart
-dart ../dart_mcp/example/dash_client.dart --server bin/main.exe
-```
-
-- If you want the client to invoke server tools without asking for human approval,
-add the `--auto` flag.
-- If you want the client to output the raw LLM responses to the CLI instead of
-re-writing responses in the tone of Dash, add the `--raw` flag.
-
 ### With the example WorkflowBot
 
-After compiling the binary, you can run the example dash chat bot to interact
-with it, from the root of this package, after compiling the binary, run:
+After compiling the binary, you can run the example workflow chat bot to
+interact with it, from the root of this package, after compiling the binary
```

Comment:
nit: 

After compiling the MCP server binary, you can run the example workflow chat bot to
interact with the server. Note that the workflow bot sets...

`<command here>`